packages/glibmm_2_4.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Glibmm_2_4 < Package
+class Glibmm_2_4 < Meson
   description 'C++ bindings for GLib'
   homepage 'https://www.gtkmm.org'
   version '2.66.5'
```

Comment:
Update to [2.78.1](https://gitlab.gnome.org/GNOME/glibmm/-/tags/2.78.1)?

Reply:
That's in glibmm_2_68...